web/openapi/openapi.yaml

```diff
@@ -50,6 +50,20 @@ components:
           $ref: "#/components/schemas/PipelineAutostart"
         autodelete:
           $ref: "#/components/schemas/PipelineAutodelete"
+        max_retries:
+          type: integer
+          description: If `autostart.failed` is `true`, the maximum amount of retries
+            for a failed pipeline.
+          default: 10
+          example: 15
```

Comment:
Hmm... for creation, `max_retries` can be an optional parameter, and only if an explicit `max_retries` parameter is provided, the amount of retries will be set to that finite value.

For updating, `max_retries` could be set to zero or below zero, which would mean "disabling" the limit. Maybe these semantics can be applied to creation, too.

This is considering the `max_retries` parameters are still `integer`s; turning them into `string`s would allow for explicit `none` values.

Reply:
As per our discussion: Nobody actually needs the `max_retries` parameter right now. I removed it.